packages/@n8n/task-runner/src/config/sentry-config.ts

```diff
@@ -0,0 +1,21 @@
+import { Config, Env } from '@n8n/config';
+
+@Config
+export class SentryConfig {
+	/** Sentry DSN */
+	@Env('N8N_SENTRY_DSN')
+	sentryDsn: string = '';
+
+	//#region Metadata about the environment
+
+	@Env('N8N_VERSION')
+	n8nVersion: string = '';
+
+	@Env('ENVIRONMENT')
+	environment: string = '';
+
+	@Env('DEPLOYMENT_NAME')
+	deploymentName: string = '';
```

Comment:
Should we populate this ourselves based on the task runner name and ID instead of exposing it to the user?

Reply:
These are the same environment variables we use in the n8n app as well. They are basically metadata about the deployment for sentry